spec/lib/dor/text_extraction/ocr_spec.rb

```diff
@@ -192,7 +192,7 @@ def build_file(sdr_peserve, filename)
         FileUtils.mkdir_p(ocr.abbyy_output_path)
         allow(Honeybadger).to receive(:notify)
         count = 0
-        allow(FileUtils).to receive(:rm_r) do
+        allow(ocr).to receive(:`) do
```

Comment:
ha, neat, didn't realize this was the method name for the shell-out-via-backtick syntax, good to know you can intercept the call like this.

Reply:
I didn't either until I asked github copilot and the all knowing AI told me